<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>自定义游戏</string>
  </property>
  <widget class="QLabel" name="labelRow">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入行数(8~99)</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCol">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入列数(8~99)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxRow">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>43</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCol">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>43</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMine">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入雷数</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMine">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>43</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
